src/style.css

```diff
@@ -1 +1,61 @@
-/* Styles go here */
+.header {
+  display: flex;
+  align-items: center;
+  padding: 10px 0;
+  max-width: 1200px;
+  font-family: Roboto, sans-serif;
+  justify-content: space-between;
+}
+
+.header__logo {
+  margin: 0 0 0 50px;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  margin-right: 50px;
```

Comment:
remove margin for nav, add padding to header

Reply:
I've fixed problems and made "npm run deploy" and also i changed gap with margin-right cause when i rum npm test, it doesn't see this indentation.
Sorry for my bad English, i can write on ukrainian or russian if it's need)